docker-compose.yml

```diff
@@ -7,8 +7,12 @@ services:
     networks:
       - back-tier
   web:
-    image: markstory/lint-review
-    command: 'gunicorn -c /code/settings.py lintreview.web:app'
+    build: .
```

Comment:
I think having a local dev default is reasonable. People running docker-compose in prod can likely figure out how to use images.